test/harness/index.js

```diff
@@ -140,7 +151,7 @@ function assert_invalid(bytes) {
 const assert_malformed = assert_invalid;
 
 function assert_soft_invalid(bytes) {
-    test(() => {
+    uniqueTest(() => {
```

Comment:
What about switching the order of the arguments to `uniqueTest`? I'd find it easier to read if the test name came before the code for the test.